compiler/src/main/scala/org/scalajs/nscplugin/JSPrimitives.scala

```diff
@@ -123,6 +125,10 @@ abstract class JSPrimitives {
     addPrimitive(Special_wrapAsThrowable, WRAP_AS_THROWABLE)
     addPrimitive(Special_unwrapFromThrowable, UNWRAP_FROM_THROWABLE)
     addPrimitive(Special_debugger, DEBUGGER)
+
+    addPrimitive(LinkingInfo_linkTimePropertyBoolean, LINKTIME_PROPERTY_BOOLEAN)
+    addPrimitive(LinkingInfo_linkTimePropertyInt, LINKTIME_PROPERTY_INT)
+    addPrimitive(LinkingInfo_linkTimePropertyString, LINKTIME_PROPERTY_STRING)
```

Comment:
Merge those to have the same associated constant? (even though they have 3 different `Symbol`s):
```suggestion
    addPrimitive(LinkingInfo_linkTimePropertyBoolean, LINKTIME_PROPERTY)
    addPrimitive(LinkingInfo_linkTimePropertyInt, LINKTIME_PROPERTY)
    addPrimitive(LinkingInfo_linkTimePropertyString, LINKTIME_PROPERTY)
```

Reply:
Nice catch, now we get the type information of `LInkingInfo.linkTimePropertyXXX` from their trees, we can merge those constant.